gateway/src/main/kotlin/systems/ajax/malov/gateway/rest/StockRecordsController.kt

```diff
@@ -0,0 +1,47 @@
+package systems.ajax.malov.gateway.rest
+
+import jakarta.validation.Valid
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import reactor.core.publisher.Mono
+import systems.ajax.malov.gateway.client.NatsClient
+import systems.ajax.malov.gateway.dto.AggregatedStockRecordResponseDto
+import systems.ajax.malov.gateway.dto.GetBestStockSymbolsWithStockRecordsRequestDto
+import systems.ajax.malov.gateway.mapper.AggregatedStockRecordResponseDtoMapper.toAggregatedStockItemResponseDto
+import systems.ajax.malov.input.reqreply.stock.get_all_man_sym.proto.GetAllManageableStockSymbolsRequest
+import systems.ajax.malov.input.reqreply.stock.get_all_man_sym.proto.GetAllManageableStockSymbolsResponse
+import systems.ajax.malov.input.reqreply.stock.get_best_stock_symbols_with_stocks.proto.GetBestStockSymbolsWithStockRecordsRequest
+import systems.ajax.malov.input.reqreply.stock.get_best_stock_symbols_with_stocks.proto.GetBestStockSymbolsWithStockRecordsResponse
+import systems.ajax.malov.internalapi.NatsSubject
+
+@RestController
+@RequestMapping("/api/v1/stock-records")
+class StockRecordsController(private val natsClient: NatsClient) {
+
+    @GetMapping("/best")
+    fun getFiveBestStockSymbolsWithStockRecords(
+        @Valid requestDto: GetBestStockSymbolsWithStockRecordsRequestDto,
```

Comment:
do we pass `GetBestStockSymbolsWithStockRecordsRequestDto` as request body?

Reply:
No, we pass as request param